src/main/java/edu/ucsb/cs156/gauchoride/entities/Ride.java

```diff
@@ -18,13 +21,22 @@
 @NoArgsConstructor
 @Builder
 @Entity(name = "ride")
+@Table(name= "ride")
 public class Ride {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
     private String day;
-    private String studentName; 
-    private String driverName; 
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")

```

Comment:
This is my fault, they should reference different column names such as rider_id or driver_id, otherwise, it won't work. That's why your code doesn't compile rn. You should make sure your code works locally at least before creating PR.